docs/Frequently Asked Questions/React Native/index.md

```diff
@@ -0,0 +1,251 @@
+
+!!! Note
+This guide is not meant to walk you through how to make a React Native app but help you integrate SDL into an existing application. We will show you a basic example of how to communicate between your app's JavaScript code and SDL's native Obj-C code. For more advanced features, please refer to the React Native documentation linked above.
+!!!
+
+## Integration Basics
+Native API methods are not exposed automatically to JavaScript. This means you must expose methods you wish to use from SDL to your React Native app. You must implement the `RCTBridgeModule` protocol into a bridge class (see below for an example). Please follow [SmartDeviceLink Integration Basics](Getting Started/Integration Basics) for the basic setup of a native SDL `ProxyManager` class that your bridge code will communicate with. This is the necessary starting point in order to continue with this example. Also set up a simple UI with buttons and some text on the SDL side.
+
+### Creating the RCTBridge
+To create a native module you must implement the `RCTBridgeModule` protocol. Update your `ProxyManager` to include `RCTBridgeModule`.
+
+##### Objective-C
+###### ProxyManager.h
+```objc
+#import <React/RCTBridgeModule.h>
+
+@interface ProxyManager : NSObject <RCTBridgeModule>
+
+<#...#>
+
+@end
+```
+
+###### ProxyManager.m
+An `RCT_EXPORT_MODULE()` macro must be added to the implementation file to expose the class to React Native.
+```objc
+@implementation ProxyManager
+
+RCT_EXPORT_MODULE();
+<#...#>
+
+@end
+```
+
+##### Swift
+Before you move forward, you must add  `#import "React/RCTBridgeModule.h"` to your `Bridging Header`. When creating a Swift application and importing Objective-C code, Xcode should ask if it should create this header file for you. You must include this bridging header for your React Native app to work. You can create this file manually as well. 
+
+```swift
+@objc(ProxyManager)
+
+class ProxyManager: NSObject {
+
+<#...#>
+
+}
+```
+
+Next, to expose the above Swift class to React Native, you must create an Objective-C file and wrap the Swift class name in a `RCT_EXTERN_MODULE` in order to use the Swift class in a React Native app. 
+
+###### ProxyManager.m 
+```objc
+#import "React/RCTBridgeModule.h"
+
+@interface RCT_EXTERN_MODULE(ProxyManager, NSObject)
+
+@end
+```
+
+### Emitting Event Notifications to JavaScript
+Inside the `ProxyManger` class post a notification for a particular event you wish to execute. The bridge will observe this event and will call the React Native listener that you will set up later in the documentation below.
```

Comment:
```suggestion
Inside the `ProxyManger` class, post a notification for a particular event you wish to execute. The bridge will observe this event and will call the React Native listener that you will set up later in the documentation below.
```

Reply:
I don't understand:

> The bridge will observe this event and will call the React Native listener

Isn't the bridge the `ProxyManager`? It conforms to the `RCTBridgeModule` protocol above... What do you mean by "bridge"? Should you be using "Event Emitter" instead of "Bridge"? I just want to make sure that we're using different words for different concepts.